### PR TITLE
ret t2 4pc change

### PR DESCRIPTION
**Pull request by Adamrch**
change 4pc to new bonus on PTR with deterministic seal judging and double judge disabled on t2 4pc.
Leave baseline double judge on until phase 5.

Update truthbearer (to 2.0) and other items ppm.

Make scrolls and truthbearer remove the other on gain.

### CODE REVIEW

File: sim/paladin/item_sets_pve.go [hunk 1]
```diff
@@ -215,24 +216,14 @@ var ItemSetRadiantJudgement = core.NewItemSet(core.ItemSet{
 				Label: "S03 - Item - T2 - Paladin - Retribution 4P Bonus",
```
Comment (kayla-glick) [inline sim/paladin/item_sets_pve.go:216]: Would you mind updating the comment for this above here?

File: sim/paladin/item_sets_pve.go [hunk 2]
```diff
@@ -215,24 +216,14 @@ var ItemSetRadiantJudgement = core.NewItemSet(core.ItemSet{
...
+					newCD := core.Cooldown{
+						Timer:    paladin.NewTimer(),
+						Duration: paladin.judgement.CD.Duration - (5 * time.Second),
+					}
 
-						// Two of the possible implementations - TODO figure out actual implementation
-						// anySealsDifferent := sealsThisJudgement != sealsLastJudgement // Would be nice!
-						// allSealsDifferent := (int(sealsThisJudgement) & int(sealsLastJudgement)) == 0 // More conservative option
-						anySealsGained := (int(sealsThisJudgement) & ^int(sealsLastJudgement)) > 0 // More conservative option (most likely option)
+					paladin.judgement.CD = newCD
```
Comment (kayla-glick) [inline sim/paladin/item_sets_pve.go:224]: If it just reduces the cooldown by 5 seconds I think you could just do
```
paladin.judgement.CD.Duration -= time.Second * 5
```

File: sim/paladin/item_sets_pve.go (hunk 1 above)
Comment (Adamrch) [reply sim/paladin/item_sets_pve.go:216]: updated

File: sim/paladin/item_sets_pve.go (hunk 2 above)
Comment (Adamrch) [reply sim/paladin/item_sets_pve.go]: changed and tested

**Pull request merged**